jnosql-mongodb/src/test/java/org/eclipse/jnosql/databases/mongodb/communication/MongoDBSpecificFeaturesTest.java

```diff
@@ -119,10 +118,10 @@ public void shouldReturnErrorOnAggregateWhenThereIsNullParameter() {
 
     @Test
     public void shouldAggregate() {
-        List<Bson> predicates = Arrays.asList(
+        Bson[] predicates = {
                 Aggregates.match(eq("name", "Poliana")),
                 Aggregates.group("$stars", Accumulators.sum("count", 1))
-        );
+        };
```

Comment:
Could you also create a test for the new aggregate?

Reply:
I do not have dockerized test. just expect it shouldn't fail...